cfgov/jinja2/v1/ask-cfpb/landing-page-spanish.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'spanish-ask-base.html' %}
+
+{% block babel_content %}
+<section class="content full-bleed">
+    <div class="primary">                
+        <div class="ac-catnav-row">              
+        {% for cat in categories %}
+            <article class="ac-catnav ac-cat-{{ cat.slug_es }}">
+                <h2 class="ac-catnav-heading"><a href="">
+    			<img src="{{static('nemo/_/img/icon-' + cat.slug + '-retina.png') }}" alt="" class="icon-40">{{ cat.name_es }}</a></h2>
+                <ul class="ac-catnav-list unstyled spaced">
+                    {% for a in cat.featured_answers() %}
+                     <li>
+                     	<a href="{{ slugurl(a.slug_es) }}">{{ a.question_es|safe }}</a>
+                     </li>
+                    {% endfor %}
+                </ul>
+                <a href="" class="ac-catnav-learn"><img src="{{ static('nemo/_/img/icon-arrow-right-disc-retina.png')}}" alt="" class="icon-18">Más acerca de {{ cat.name_es|lower }}</a>
```

Comment:
Thanks for catching that! The category page is actually still being constructed, so I'll update the link when it's ready.